src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -576,49 +560,11 @@ def __call__(self, input_ids, attention_mask, token_type_ids, position_ids, dete
 
 
 @add_start_docstrings(
-    """
-    Bert Model with two heads on top as done during the pretraining: a `masked language modeling` head and a `next
-    sentence prediction (classification)` head.
-    """,
+    "The bare Bert Model transformer outputting raw hidden-states without any specific head on top.",
     BERT_START_DOCSTRING,
 )
-class FlaxBertForPreTraining(FlaxBertPreTrainedModel):
```

Comment:
This code is to always the same for different `flax.linen.Modules` => IMO, it's cleaner to put it into `FlaxBertPreTrainedModel` at the expense of possibly having some exceptions in the future `FlaxBertForCausalLM` that will need `past_key_values`, `encoder_attention_mask`, ... additionally.